dev/spec-setup.js

```diff
@@ -6,7 +6,7 @@ var MochaAdapter = rowdy.adapters.mocha;
 
 var SERVER_HOST = "127.0.0.1";
 var SERVER_PORT = "3000";
-var DEV_SERVER_TIMEOUT = 20000;
+var DEV_SERVER_TIMEOUT = process.env.TEST_FUNC_DEV_SERVER_TIMEOUT || 20000;
```

Comment:
GLOBAL: Prefer `npm` `config` style variables that can be overridden either by env or by `package.json:config`

See https://github.com/FormidableLabs/builder#npm-config

The result would be something (and sorry it's long) like:

```js
var DEV_SERVER_TIMEOUT = process.env.npm_package_config_test_func_dev_server_timeout || 20000;
````